packages/block-editor/src/components/use-block-drop-zone/test/index.js

```diff
@@ -130,46 +141,10 @@ describe( 'getNearestBlockIndex', () => {
 				orientation
 			);
 
-			expect( result ).toBe( 2 );
-		} );
-
-		it( 'returns `2` when the position is nearest to the start of the third block', () => {
```

Comment:
I deleted some tests here as I think they are a bit repetitive. @talldan is there something that I missed but is important in those tests?

Reply:
Reasoning about the third block (the one with the index of 2) is useful because it's within a block list that wraps into multiple columns or rows and it's the last block in its column/row. I think it'd be ok to consider reducing the test data to only three blocks instead of removing a test case.

One thing that's nice to test is a cursor position that's outside of the block rects. `'returns `3` when the position is past the end of the third block'` previously did that. It might be worth making the test data more realistic. Like I notice all the blocks are completely adjacent, which doesn't really happen much IRL. Instead users often drag into the gap between the blocks.

Reply:
I see! I added back the deleted tests in 86b13abd8e.

> One thing that's nice to test is a cursor position that's outside of the block rects.

Agreed! That's why I added some of these tests in the e2e tests below. It would be good to review them as well.

Reply:
I refactored the tests again in cce8d8f.